redisinsight/ui/src/components/database-overview/hooks/useDatabaseOverview.spec.ts

```diff
@@ -0,0 +1,239 @@
+import { cloneDeep, set } from 'lodash'
+import {
+  act,
+  initialStateDefault,
+  mockStore,
+  renderHook,
+} from 'uiSrc/utils/test-utils'
+import { getDatabaseConfigInfo } from 'uiSrc/slices/instances/instances'
+import { setConnectivityError } from 'uiSrc/slices/app/connectivity'
+import { sendEventTelemetry, TelemetryEvent } from 'uiSrc/telemetry'
+import { useDatabaseOverview } from './useDatabaseOverview'
+
+// Mock the telemetry function
+jest.mock('uiSrc/telemetry', () => ({
+  sendEventTelemetry: jest.fn(),
+  TelemetryEvent: {
+    OVERVIEW_AUTO_REFRESH_ENABLED: 'overview_auto_refresh_enabled',
+    OVERVIEW_AUTO_REFRESH_DISABLED: 'overview_auto_refresh_disabled',
+  },
+}))
```

Comment:
It's OK as is, but I'm wondering if it makes more sense to use the actual `TelemetryEvents`. In this case, if someone changes them, the test would potentially be more meaningful.
```suggestion
jest.mock('uiSrc/telemetry', () => {
  const actual = jest.requireActual('uiSrc/telemetry')
  return {
    ...actual,
    sendEventTelemetry: jest.fn(),
  }
})
```